src/assets/Assets.ts

```diff
@@ -1,13 +1,16 @@
 import _ from 'lodash';
 
-interface CustomObject {[key: string]: any}
+interface CustomObject {
+  [key: string]: any;
+}
 
 function assignProperties(a: CustomObject, b: {[key: string]: any}) {
   if (a) {
-    _(b).keys().forEach((key) => {
-      // @ts-ignore
-      Object.defineProperty(a, key, Object.getOwnPropertyDescriptor(b, key));
-    });
+    _.flow(_.keys, arr =>
+      _.forEach(arr, key => {
+        // @ts-ignore
+        Object.defineProperty(a, key, Object.getOwnPropertyDescriptor(b, key));
+      }))(b);
```

Comment:
Why did you change here if there's no _.chain?

Reply:
It's an unexplicit chain - https://lodash.com/docs/4.17.15#prototype-chain

Reply:
Wouldn't it be enough to just change it to `_.keys(b).forEach`? Would that work? 
I personally find the _.flow not very readable, so whenever possible, I'd use a different approach. 

In this case we can also use vanilla JS and do `Object.keys(b).forEach`

Reply:
Done.
I changed to vanilla JS where I found it more readable and also renamed the parameters of the methods as Miki suggested.